### ISSUE TITLE
Support for get_block_header_batch 

### ISSUE THREAD

**Issue by bilthon** [bitshares/bitshares-core#262]
The ```get_block_header``` database API call now accepts as an argument a single uint32_t value specifying the block number.

It would be nice for some applications if we could ask for a batch of block headers in a random way. 

Something like this:

```get_block_header_batch(const vector<uint32_t block_num>)```

The return for this call could be exactly the same as the ```get_block_header``` API call, that is the block data or null if no matching block was found for that array position, but respecting the ordering specified in the request, so as to allow the client to map between requested and given block header data.

**Comment by oxarbitrage** [comment]
https://github.com/bitshares/bitshares-core/pull/263

sample usage:

```
root@alfredo:~/bitshares-munich/issue262/bitshares-core# curl --data '{"jsonrpc": "2.0", "params": ["database", "get_block_header_batch", [[15794002,15794000]]],
 "method": "call", "id": 10}' http://23.94.69.140:8090/rpc | jq
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   549  100   430  100   119   471k   130k --:--:-- --:--:-- --:--:--  419k
{
  "id": 10,
  "result": [
    [
      15794000,
      {
        "previous": "00f0ff4fdb5492b12818b143428a6ed67232364b",
        "timestamp": "2017-04-18T18:27:48",
        "witness": "1.6.16",
        "transaction_merkle_root": "7c22f2e2850ab404f06c7d5c007ccb4a06c232bc",
        "extensions": []
      }
    ],
    [
      15794002,
      {
        "previous": "00f0ff51267edf46d31e23d955fa55e773e84f2f",
        "timestamp": "2017-04-18T18:27:54",
        "witness": "1.6.27",
        "transaction_merkle_root": "bb2fb2aedda730ea28ffa088a77d1fcf60e873d1",
        "extensions": []
      }
    ]
  ]
}
root@alfredo:~/bitshares-munich/issue262/bitshares-core# 
```

**Comment by vikramrajkumar** [comment]
Merged in #263 above.
